### PR TITLE
Lid for a bucket

### PR DESCRIPTION
**Pull request by MrRomainzZ**
## What Does This PR Do
It adds a texture and a code to a closed bucket

## Why It's Good For The Game
You can see when bucket is closed

## Images of changes
These buckets to be exact
dreamseeker_E4hnJEZcRI

## Testing
Tested these changes in my local game

## Changelog
:cl:
tweak: Buckets now show their lid
/:cl:

### CODE REVIEW

**Comment by CinnamonSnowball** [comment]
The second part of this PR would be a partial revert of #20613

**Comment by MrRomainzZ** [comment]
For combat options as far as I see. Let me then undo that

File: code/modules/reagents/reagent_containers/glass_containers.dm [hunk 1]
```diff
@@ -307,6 +307,11 @@
 	container_type = OPENCONTAINER
 	dog_fashion = /datum/dog_fashion/head/bucket
 
+/obj/item/reagent_containers/glass/bucket/update_overlays()
+	. = ..()
+	if(!is_open_container())
+		. += "lid_[initial(icon_state)]"
```
Comment (DGamerL) [inline code/modules/reagents/reagent_containers/glass_containers.dm:313]: Isn't this already handled in `glass_containers.dm`, line 166 to 167?

**Review by DGamerL — CHANGES_REQUESTED** [review]

File: code/modules/reagents/reagent_containers/glass_containers.dm (hunk 1 above)
Comment (MrRomainzZ) [reply code/modules/reagents/reagent_containers/glass_containers.dm:313]: It is but only for beakers

**Review by DGamerL — APPROVED** [review]
I am silly and that file is aswell, my bad

**Review by lewcc — APPROVED** [review]

**Pull request merged**